modules/scanning-images-by-using-secured-clusters.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * operating/examine-images-for-vulnerabilities.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="scanning-images-by-using-secured-clusters_{context}"]
+= Scanning images by using secured clusters
+
+By configuring delegated image scanning, you can scan images by using the secured clusters instead of the Central services.
+
+A new delegated registry configuration specifies the registries from which you can delegate image scans. For images that Sensor observes, you can use the delegated registry configuration to delegate scans from no registries, all registries, or specific registries. 
```

Comment:
"Newly delegated" if "delegated" is a descriptor of yours. "New delegated" is fine if the feature name is "delegated registry configuration."